### PR TITLE
files for containerization

### PR DESCRIPTION
**Pull request by Grant-Steinfeld**
@max-shapiro32 please review and merge if OK
thanks!

### CODE REVIEW

File: src/currencyexchange/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (moficodes) [inline src/currencyexchange/manifest.yml]: Do we care for cloud foundry?

File: src/currencyexchange/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:10.16.1-alpine
```
Comment (moficodes) [inline src/currencyexchange/Dockerfile:1]: this is fine for now. (I used something similar for destination as well)

We will need to optimize the image more later.

**Review by moficodes — APPROVED** [review]

File: src/currencyexchange/manifest.yml (hunk 1 above)
Comment (MaxShapiro32) [reply src/currencyexchange/manifest.yml]: @Grant-Steinfeld I thought this was supposed to be deployed to k8s?

**Review by MaxShapiro32 — CHANGES_REQUESTED** [review]
Include the k8s files in the config repo too

**Review by MaxShapiro32 — APPROVED** [review]
This k8s related files will need to be put here: https://github.com/bee-travels/config

So as a result, this pr is fine

**Pull request merged**